### ISSUE TITLE
Any way to show the address/offset?

### ISSUE THREAD

**Issue by Anthony-J-Garot** [mvduin/bbb-pin-utils#6]
This is a great tool for seeing what I have configured if I know the pin name. Thanks for writing it.

When I use the SysConfig tool at TI, I work with the address and offset.

So, for example, P9_31 corresponds to 0x44e10990 (or 0x990 shorthand). I get these from header PDFs from Derek Molloy.

Having the shorthand address in your tool would make design easier and double-checking the .dtsi file much easier.

**Comment by mvduin** [comment]
The pin numbers used by my tools are the index into the pin config array located at 0x44e10800, so for example show-pin lists P9.31 as pin 100 therefore its pinmux register is at 0x44e10800 + 100 * 4 which is indeed 0x44e10990.

**Comment by Anthony-J-Garot** [comment]
It would be nice to see 0x990 at a glance and not have to do math to convert.

**Comment by mvduin** [comment]
You could patch the printf that prints the output from
```perl
	printf "%-32s $Y{'%3s'} %s\n", $label, $pin, $function;
```
to
```perl
	printf "%-32s $Y{'%03x'} %s\n", $label, 0x800+4*$pin, $function;
```

**Comment by mvduin** [comment]
I guess I could add a commandline flag for it

**Comment by Anthony-J-Garot** [comment]
👍  Because so much is shown already, a command line flag makes sense.

**Comment by mvduin** [comment]
Fixed by commit f738124
